src/Rap2hpoutre/LaravelLogViewer/LaravelLogViewer.php

```diff
@@ -81,20 +81,26 @@ public function setFile($file)
      */
     public function pathToLogFile($file)
     {
-        $logsPath = $this->storage_path;
-        $logsPath .= ($this->folder) ? '/' . $this->folder : '';
-
         if (app('files')->exists($file)) { // try the absolute path
             return $file;
         }
-
-        $file = $logsPath . '/' . $file;
-
-        // check if requested file is really in the logs directory
-        if (dirname($file) !== $logsPath) {
-            throw new \Exception('No such log file');
+        if (is_array($this->storage_path)) {
+            foreach ($this->storage_path as $value) {
+                if (app('files')->exists($value . '/' . $file)) { // try the absolute path
```

Comment:
@rap2hpoutre with $value . '/' . $file the code will verify if the file (with the folder associate) exists (not a random one) $value is equal to the folder. Maybe rename the variable to be more clear. Also I just fix some issue with the view and the content when have multiple location. 
`<a href="?l={{ \Illuminate\Support\Facades\Crypt::encrypt($file) }}` should be 
`<a href="?l={{ \Illuminate\Support\Facades\Crypt::encrypt($folder."/".$file) }}`
I will push the changes and we can discuss after